terminal_ui.py

```diff
@@ -0,0 +1,46 @@
+from game import Game
+
+
+class TerminalUI:
+    def __init__(self):
+        self.play()
+
+    def play(self):
+        game.set_up_game()
+        self.print_board()
+        while not game.winner:
+            self.make_move()
+        print('You won!')
+
+    def make_move(self):
+        self.print_score()
+        row, column = self.ask_for_move()
+        game.play_game(row, column)
+        self.print_board()
+        if game.reset_board:
```

Comment:
shouldn't this condition be checked inside the game class, maybe as part of play_game.
If this is the same as losing, then this should be moved to the place where we check the winner

Reply:
In the game class reset_board states that the current board doesn't have anymore possible moves and that the board has to be refreshed to keep playing. I have to check it to see if the board needs to be refreshed. But I changed it a bit so that this condition is the result of calling the game.play_game function. Is this better?